ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -13,6 +13,7 @@ interface ListingCardTableProps extends StandardTableProps, StackedTableProps {}
 
 export interface CardHeader {
   customClass?: string
+  element?: React.ReactNode
   text: string
```

Comment:
should `text` now be optional?

Reply:
I think we can have one prop called `content` where the type is `string | React.ReactNode`

Reply:
is it okay to do a breaking change? otherwise, we could awkwardly leave the prop name as `text` 🥴

Reply:
Yep it's not a problem!